pkg/b2e2ec84-841b-4d30-b031-76324d8ac8f8/package.lp

```diff
@@ -1,14 +1,14 @@
 (librepcb_package b2e2ec84-841b-4d30-b031-76324d8ac8f8
- (name "LibrePCB Logo")
+ (name "LIBREPCB_LOGO")
```

Comment:
I wonder if we should invert this, to have a shared `LOGO_`-prefix (i.e. `LOGO_LIBREPCB`)?

This would be in line with other package names, where the type comes first, followed by the specification / dimensions.

Same thing for CE, of course.

Reply:
Sounds reasonable! :+1: I renamed all three packages.